docker/src/analyze_package.R

```diff
@@ -45,6 +45,19 @@ getHyperlink <- function(analyzed_file){
     gh_link <- paste0("[",file_ref,"](",file_hyperlink,")")
 }
 
+# this function generates a markdown version of an input list
+getInputsMarkdown <- function(inputList){
+    markdown_res <- ""
+    for (i in seq(length(inputList))){
+        name <- names(inputList)[i]
+        value <- inputList[i]
+        markdown_res <- paste0(markdown_res,"<details><summary>",name,"</summary>")
+        markdown_res <- paste0(markdown_res,value,"</details>")
+    }
+
+    return(markdown_res)
```

Comment:
please avoid return at end of function (it is not necessary in R unless you are returning early, before the end of function).